app/libs/useId.js

```diff
@@ -0,0 +1,5 @@
+export const useId = (req, param) => {
```

Comment:
Here you should use default export, just because file name useId and this function useId - main